Makefile

```diff
@@ -37,10 +37,12 @@ LINT=golangci-lint
 MISSPELL=misspell -error
 MISSPELL_CORRECTION=misspell -w
 
-BUILD_INFO_IMPORT_PATH=github.com/signalfx/splunk-otel-collector/internal/version
+BUILD_INFO_IMPORT_PATH=go.opentelemetry.io/collector/internal/version
```

Comment:
Should we keep `github.com/signalfx/splunk-otel-collector/internal/version`, otherwise how will the version get set for https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/main.go#L83?

Reply:
Right. I think it should be 

```
BUILD_INFO_IMPORT_PATH=github.com/signalfx/splunk-otel-collector/internal/version
BUILD_INFO_IMPORT_PATH_CORE=go.opentelemetry.io/collector/internal/version
```

I didn't notice that it was replacement for splunk distro. Now it makes sense why it didn't work before

Reply:
> Right. I think it should be
> 
> ```
> BUILD_INFO_IMPORT_PATH=github.com/signalfx/splunk-otel-collector/internal/version
> BUILD_INFO_IMPORT_PATH_CORE=go.opentelemetry.io/collector/internal/version
> ```
> 
> I didn't notice that it was replacement for splunk distro. Now it makes sense why it didn't work before

Sorry, my bad, it was actually pointing to the version in our distro when I tested it but for the PR somehow I replaced that line, thanks for catching that!